lib/signaling/v2/room.js

```diff
@@ -184,7 +213,26 @@ RoomV2.prototype._update = function _update(roomState) {
  * @returns {Promise.<Array<StatsReport>>}
  */
 RoomV2.prototype.getStats = function getStats() {
-  return this._peerConnectionManager.getStats();
+  var self = this;
+
+  function filterLocalTrackStats(trackStats) {
+    return self._published.has(trackStats.trackId);
```

Comment:
@markandrus Can't we do this instead?
```javascript
const publishedTracks = flatMap(this.localParticipant.tracks).filter(track => !!track.sid);
return publishedTracks.has(trackStats.trackId);
```

Reply:
The `localParticipant.tracks` and RemoteParticipant `tracks` may not include the full set of Tracks, since we will remove from those collections as Tracks are removed.